Source/WebKit/UIProcess/API/glib/WebKitWebContext.cpp

```diff
@@ -417,11 +409,12 @@ static void webkitWebContextConstructed(GObject* object)
     API::ProcessPoolConfiguration configuration;
     configuration.setInjectedBundlePath(FileSystem::stringFromFileSystemRepresentation(bundleFilename.get()));
     configuration.setUsesWebProcessCache(true);
-#if PLATFORM(GTK)
+#if PLATFORM(GTK) && !USE(GTK4)
     configuration.setProcessSwapsOnNavigation(priv->psonEnabled);
-#if !USE(GTK4)
     configuration.setUseSystemAppearanceForScrollbars(priv->useSystemAppearanceForScrollbars);
-#endif
+#else
+    configuration.setProcessSwapsOnNavigation(true);
+    configuration.setUseSystemAppearanceForScrollbars(false);
```

Comment:
This broke the wpe build, setUseSystemAppearanceForScrollbars is GTK specific

Reply:
Sigh, fixed....